content/docs/user-guide/programming/components/editor-components.md

```diff
@@ -71,9 +72,9 @@ The code for editor components is similar to the code for runtime components. Th
 
```

Comment:
In the H2 (##) section header on line 69, change "Editor Component and Runtime Component Differences" to use sentence casing.